libraries/Microsoft.Bot.Builder/TelemetryConstants.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder
+{
+    public static class TelemetryConstants
+    {
+        public const string ChannelIdProperty = "channelId";
```

Comment:
Should all telemetry properties follow the same casing? This middleware & the luis/qna samples use lower camel case. The waterfall telemetry & initializer is upper camel case.

Reply:
Argh.  
The Initializer properties begin with lowercase, so we're already inconsistent.  I'm leaning towards aligning with Initializer and begin with lowercase character - any thoughts?

Tempted to lowercase waterfall telemetry, but that's actually a breaking change.

---
In reply to: [263166401](https://github.com/Microsoft/botbuilder-dotnet/pull/1420#discussion_r263166401) [](ancestors = 263166401)

Reply:
I agree with lowercase